app.js

```diff
@@ -26,7 +26,7 @@ app.post('/verify/:platform', async (req, res) => {
     const verifiedResult = await verifier.verify(accountId, handle, proof, challenge);
     let verified = null;
 
-    if (platform === "twitter") {
+    if (platform === "twitter" || "google") {
```

Comment:
This if statement always evaluates to `true` because the `OR` operator is not working as expected. I think you meant `if (platform === "twitter" || platform === "google")`. Anyways, I recommend set a new global constant on the top of this file `const SPECIAL_PLATFORMS = ["twitter", "google"];` and use `if (SPECIAL_PLATFORMS.includes(platform))`.